packages/eslint-config-finn-prettier/react.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = {
+    extends: [
+        require.resolve('./prettier.js'),
+        'prettier/react',
```

Comment:
doesn't this order mean that `prettier/react` overrides the definitions in `./prettier.js`?